airflow/api/auth/backend/kerberos_auth.py

```diff
@@ -55,7 +55,7 @@ def init_app(app):
 
     hostname = app.config.get('SERVER_NAME')
     if not hostname:
-        hostname = getfqdn()
+        hostname = get_hostname()
```

Comment:
I don’t think this will work for Kerberos. If gethostbyname returns something else than get_hostname that won’t work. As socket.fqdn uses gethostbyname I don’t think this makes sense

Reply:
Thanks, I was unsure about this one, primarily because it technically **could** be wanted, but would require quite a bit of an odd krb setup if you ask me. Removed from diff.

Reply:
The reason why I had left it in is purely for consistency in the end. Ie, if you tell airflow, **use this hostname instead**, then I feel it should always use said hostname. I have reverted this however.